custom_components/waste_collection_schedule/waste_collection_schedule/source/midandeastantrim_gov_uk.py

```diff
@@ -0,0 +1,118 @@
+from datetime import date, datetime
+
+import requests
+from bs4 import BeautifulSoup, Tag
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Mid and East Antrim"
+DESCRIPTION = "Source for Mid and East Antrim Borough Council."
+URL = "https://www.midandeastantrim.gov.uk"
+TEST_CASES = {
+    "185438838": {"uprn": 185438838},
+    "185448385": {"uprn": "185448385"},
+    "187262293": {"uprn": 187262293},
+}
+
+
+ICON_MAP = {
+    "grey": "mdi:trash-can",
+    "glass": "mdi:bottle-soda",
+    "green": "mdi:leaf",
+    "blue": "mdi:recycle",
+}
```

Comment:
This seems not to match your collection types, so all collections have icon None

Reply:
Thanks for pointing out, have updated now